.github/helpers/check-npm-dependency-versions.js

```diff
@@ -0,0 +1,170 @@
+const https = require('https');
+const path = require('path');
+
+// Get package.json file.
+const cmdLineArgument = process.argv.slice(2)[0];
+const packageJsonPath = cmdLineArgument.startsWith('./')
+  ? path.resolve(cmdLineArgument)
+  : path.resolve(`./${cmdLineArgument}`);
+const packageJson = require(packageJsonPath);
+
+// Read the package.json file and get the list of dependencies and devDependencies.
+const dependencies = Object.entries(packageJson.dependencies) ?? undefined;
+const devDependencies = Object.entries(packageJson.devDependencies) ?? undefined;
+
+// Results from running checkVersions.
+let dependencyResults = [];
+let devDependencyResults = [];
```

Comment:
Should be using `const` here...? We don't typically mutate arrays.

Reply:
Good point. Will change.

Reply:
No they change.